frontend/packages/git-service/src/services/gitlab-service.ts

```diff
@@ -157,6 +157,33 @@ export class GitlabService extends BaseService {
     }
   };
 
+  isDevfilePresent = async (): Promise<boolean> => {
+    try {
+      const projectID = await this.getProjectId();
+      await this.client.RepositoryFiles.showRaw(
+        projectID,
+        'devfile.yaml',
+        this.metadata.defaultBranch,
+      );
+      return true;
+    } catch (e) {
+      return false;
```

Comment:
will it make sense to log an err as we are passing it to the caller, wdyt?

Reply:
Interesting, we don't log the value on any of the other methods 🤔  Probably should though...

Reply:
I opted out of this, I don't want this PR blooming anymore, this is existing functionality and we were okay with it... keeping it consistent for now.